src/app/modules/main/profile_section/advanced/controller.nim

```diff
@@ -122,7 +122,7 @@ proc isNimbusProxyEnabled*(self: Controller): bool =
 proc toggleNimbusProxy*(self: Controller) =
   let enabled = self.nodeConfigurationService.isNimbusProxyEnabled()
 
-  if(not self.nodeConfigurationService.setNimbusProxyConfig(not enabled)):
+  if not self.nodeConfigurationService.setNimbusProxyConfigEnabled(not enabled):
```

Comment:
Is this an anti pattern? `setNimbusProxyConfigEnabled` is a setter but returns a boolean :-o ?